conversational-skills/README.md

```diff
@@ -0,0 +1,304 @@
+--header 'Content-Type: application/json' \
+--header 'Authorization: {{AUTHORIZATION}}' \
+--data '{
+    "provider_id": "myProCodeProvider-noSpaces_no_special_chars",
+    "specification": {
+        "servers": [
+            {
+                "url": "https://myProCodeProvider.com"
+            }
+        ],
+        "components": {
+            "securitySchemes": {
+                "authentication_method": "api_key",
+            }
+        }
+    },
+    "private": {
+        "authentication": {
+            "api_key": {
+                "My-API-Key": {
+                    "type": "value",
+                    "in": "header",
+                    "value": "myAPIKeyAuth"
+                }
+            }
+        }
+    }
+}'
+```
+
+**Example Request Body**
+```json
+{
+    /* A user-provided ID that should not have spaces, tabs, or special characters. */ 
+    "provider_id": "myProCodeProvider-noSpaces_no_special_chars",
+    "specification": {
+        "servers": [
+            {
+                /* URL where the client-implemented endpoints are hosted. */ 
+                "url": "https://myProCodeProvider.com"
+            }
+        ],
+        "components": {
+            "securitySchemes": {
+                /* Required enum that defines the type of authentication needed. */
+                "authentication_method": "api_key",
+            }
+        }
+    },
+    /* Sensitive, private data for authentication. */
+    "private": {
+        "authentication": {
+            "api_key": {
+                "My-API-Key": {
+                    "type": "value",
+                    "in": "header",
+                    "value": "myAPIKeyAuth"
+                }
+            }
+        }
+    }
+}
+```
+
+**Example Response**
+
+```json
+{
+  "provider_id": "myProCodeProvider-noSpaces_no_special_chars",
+  "specification": {
+    "servers": [
+      {
+        "url": "https://myProCodeProvider.com"
+      }
+    ],
+    "components": {
+      "securitySchemes": {
+        "authentication_method": "api_key"
+      }
+    }
+  }
+}
+```
+
+- `provider_id` is immutable. A new value passed in through the request body will not be accepted by the API.
+- Information within the `private` object will not be returned by the API on response.
+- `authentication_method`: An required enum in the request body denoting the authentication method for the conversational skill provider.
+  - Accepted values today include: 'basic', 'bearer', 'api_key', 'oauth2', or 'none'
+- `private.authentication` should be an empty object when `authentication_method` is `none`.
+
+
+### Create a Skill-Backed Action
+
+Once a provider has been registered on a watsonx Assistant instance, skill-backed actions may be created from the Assistant UI. Builders may tune the logic, routing, conditions, and conversational inputs from the user for invoking the skill. 
+
+To create a skill-backed action from the API, please reference the cloud API Docs.
```

Comment:
provide link

Reply:
Link isn't there yet, will be a follow up PR where we'll hyper link all references to cloud docs